packages/manager/apps/pci-volume-backup/src/helpers/index.ts

```diff
@@ -0,0 +1,39 @@
+import { PaginationState } from '@ovh-ux/manager-react-components';
+import { ColumnSort } from '@tanstack/react-table';
+
+export const paginateResults = <T>(
+  items: T[],
+  pagination: PaginationState,
+) => ({
+  rows: items.slice(
+    pagination.pageIndex * pagination.pageSize,
+    (pagination.pageIndex + 1) * pagination.pageSize,
+  ),
+  pageCount: Math.ceil(items.length / pagination.pageSize),
+  totalRows: items.length,
+});
+
+export const compareFunction = <T>(key: keyof T) => (a: T, b: T) => {
```

Comment:
These 2 functions `compareFunction` and `sortResults` do not seem to be used in the application. Maybe we should remove them?